airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/AirbyteDebeziumHandler.java

```diff
@@ -108,10 +110,6 @@ public AutoCloseableIterator<AirbyteMessage> getSnapshotIterators(
             .fromIterator(MoreIterators.singletonIteratorFromSupplier(cdcStateHandler::saveStateAfterCompletionOfSnapshotOfNewStreams)));
   }
 
-  /**
-   * In the default case here, we don't know for sure whether the Debezium Engine will produce records
-   * or not. We therefore pass {@link canShortCircuitDebeziumEngine} = false.
-   */
```

Comment:
I'm getting rid of the `canShortCircuitDebeziumEngine` functionality. It's not easily portable and there are easier ways to speed up the tests almost as well. Namely, heartbeats and a shorter `subsequentRecordWaitTime`.